examples/sample_manifest_extended.json

```diff
@@ -0,0 +1,153 @@
+    
+    "modified": "2016-08-11T19:32:18Z",
+
+    "belongs_to": {
+      "collection": "Math for All",
+      "series": {
+        "name": "Geometry Trilogy",
+        "position": 4
+      }
+    }    
+    
+    "duration": 15153,
+    "type": "audio/mpeg",
+    "bitrate": 128,
+    "samplerate": "22KHz",
+    "channels": 2,
+    
+    "contributor": [
+        {
+        "name": {
+            "ru": "Эдвин Эбботт Эбботт",
+            "en": "Edwin Abbott Abbott"
+        },        
+        "sort_as": {
+            "ru": "Эбботт, Эдвин Эбботт",
+            "en": "Abbott, Edwin Abbott"
+        },        
+        "scheme": "https://www.loc.gov/marc/relators/",
+        "role": ["aut"]
+        }, 
+        {
+        "name": "Edwin Abbott Abbott",
+        "scheme": "https://www.loc.gov/marc/relators/",
+        "role": ["led"]
+        },
+        {
+        "name": "Abe Narrator",
+        "scheme": "https://www.loc.gov/marc/relators/",
+        "role": ["nrt"]
+        },
+        {
+        "name": "Beatrice Narrator",
+        "scheme": "https://www.loc.gov/marc/relators/",
+        "role": ["nrt"]
+        }
+    ],
+
+    "schema:hasPart": [
+        {
+            "@type": "http://schema.org/VisualArtwork",
+            "schema:url": "cover.jpg",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        },
+        {
+            "@type": "http://schema.org/Book",
+            "schema:url": "this_is_for_copyright_on_text_copy",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        },
+        {
+            "@type": "http://bib.schema.org/Audiobook",
+            "schema:url": "this_is_for_copyright_on_performance",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        }
+    ]
+
+    "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+  },
+
+  "links": [
+    {"rel": "self", "href": "https://github.com/HadrienGardeur/audiobook-manifest/blob/master/examples/flatland.audiobook-manifest", "type": "application/audiobook+json"},
+
+    {"rel": "alternate", "href": "https://hadriengardeur.github.io/webpub-manifest/examples/MobyDick/manifest.json", "type": "application/webpub+json"},
+
+    {"rel": "license", "href": "http://example.org/license/35d9b2d6", "type": "application/vnd.readium.lcp.license.v1.0+json"},
+    {"rel": "register", "href": "http://example.org/license/35d9b2d6/register{?id,name}", "type": "application/vnd.readium.license.status.v1.0+json", "templated": true},
+    {"rel": "return", "href": "http://example.org/license/35d9b2d6/return{?id,name}", "type": "application/vnd.readium.lcp.license.v1.0+json", "templated": true},
+    {"rel": "renew", "href": "http://example.org/license/35d9b2d6/renew", "type": "text/html"},
+    {"rel": "renew", "href": "http://example.org/license/35d9b2d6/renew{?end,id,name}", "type": "application/vnd.readium.lcp.license.v1.0+json", "templated": true},
```

Comment:
I'm not sure why there's a link to an LCP license and no information about the files being encrypted.

Also, the other links wouldn't show up directly in the manifest with LCP.